src/runtime/security/security_types.cpp

```diff
@@ -275,6 +275,5 @@ void negotiation_response::printTo(std::ostream &out) const
         << "msg=" << to_string(msg);
     out << ")";
 }
-
-} // namespace security
-} // namespace dsn
+}
```

Comment:
namespace should be kept

Reply:
src/runtime/security/security_types.h and src/runtime/security/security_types.cpp are auto-generated by thirft, it will be removed each time when we update thift sturctures, the namespace maybe manually added before. Should we add it manually when we change thrift sturctures?

Reply:
That's OK, not needed.